static/themes/SSF/style.css

```diff
@@ -46,3 +46,9 @@
     background-image: url(../../img/sunflower_spin_throbber.gif);
     background-repeat: no-repeat;
 }
+
+/* Comments */
+
+#comment-form {
+    width: 100%;
```

Comment:
@hitesh96db can I work on this? http://sunflower.sahanafoundation.org/eden/project/task/278

100% is looking good in smaller width screens. Should I change to 50%?